internal/kafka/internal/presenters/cluster.go

```diff
@@ -33,14 +54,75 @@ func PresentEnterpriseClusterWithAddonParams(cluster api.Cluster, fleetShardPara
 	return c, nil
 }
 
-func PresentEnterpriseCluster(cluster api.Cluster) public.EnterpriseCluster {
+func PresentEnterpriseCluster(cluster api.Cluster, consumedStreamingUnitsInTheCluster int32, kafkaConfig *config.KafkaConfig) public.EnterpriseCluster {
 	reference := PresentReference(cluster.ClusterID, cluster)
-	return public.EnterpriseCluster{
+	presentedCluster := public.EnterpriseCluster{
 		Id:                            cluster.ClusterID,
 		Status:                        cluster.Status.String(),
 		ClusterId:                     cluster.ClusterID,
 		Kind:                          reference.Kind,
 		Href:                          reference.Href,
+		CloudProvider:                 cluster.CloudProvider,
+		Region:                        cluster.Region,
+		MultiAz:                       cluster.MultiAZ,
 		AccessKafkasViaPrivateNetwork: cluster.AccessKafkasViaPrivateNetwork,
+		SupportedInstanceTypes:        public.SupportedKafkaInstanceTypesList{},
+	}
+
+	// enterprise clusters only supports standard instance type for now. It is safe to hardcode this.
+	storedCapacityInfo, ok := cluster.RetrieveDynamicCapacityInfo()[types.STANDARD.String()]
+	if ok {
+		presentedCluster.CapacityInformation = presentEnterpriseClusterCapacityInfo(consumedStreamingUnitsInTheCluster, storedCapacityInfo)
+		presentedCluster.SupportedInstanceTypes = presentEnterpriseClusterSupportedInstanceTypes(presentedCluster.CapacityInformation.RemainingKafkaStreamingUnits, kafkaConfig)
+	}
+
+	return presentedCluster
+}
+
+func presentEnterpriseClusterCapacityInfo(consumedStreamingUnitsInTheCluster int32, storedCapacityInfo api.DynamicCapacityInfo) public.EnterpriseClusterAllOfCapacityInformation {
+	return public.EnterpriseClusterAllOfCapacityInformation{
+		ConsumedKafkaStreamingUnits:  consumedStreamingUnitsInTheCluster,
+		KafkaMachinePoolNodeCount:    storedCapacityInfo.MaxNodes,
+		RemainingKafkaStreamingUnits: storedCapacityInfo.MaxUnits - consumedStreamingUnitsInTheCluster,
+		MaximumKafkaStreamingUnits:   storedCapacityInfo.MaxUnits,
+	}
+}
+
+func presentEnterpriseClusterSupportedInstanceTypes(remainingCapacity int32, kafkaConfig *config.KafkaConfig) public.SupportedKafkaInstanceTypesList {
+	// enterprise clusters only supports standard instance type for now. It is safe to hardcode this.
+	standardInstanceType, err := kafkaConfig.SupportedInstanceTypes.Configuration.GetKafkaInstanceTypeByID(types.STANDARD.String())
+	if err != nil { // this should never happen, lets log an error in case it happens.
+		logger.Logger.Errorf("failed to find standard instance type from supported instance type config due to %q.", err.Error())
```

Comment:
are we generally not returning errors from presenter functions? This hides the error, although it should be quite rare here.

Reply:
We could return error from the presenter.
My thinking here was - this error will never occur. If it occurs, then it is a configuration error. 
Let's log it to sentry and we can action it.

Reply:
Both things can be performed